testing/xfpga/test_open_close_c.cpp

```diff
@@ -333,7 +335,14 @@ INSTANTIATE_TEST_CASE_P(openclose_c, openclose_c_p, ::testing::ValuesIn(test_pla
  *             but driver is not loaded. the function returns FPGA_NO_DRIVER.
  *
  */
-TEST(openclose_c, invalid_open_close) {
+class openclose_c_mock_p
+    : public ::testing::TestWithParam<std::string> {
+ protected:
+  openclose_c_mock_p() {}
+
+};
```

Comment:
You probably want to derive from the fixture defined above (openclose_c_p)
```c++
class openclose_c_mock_p : public openclose_c_p {
};
```

Reply:
To my understanding, I don't believe we want to derive from the openclose_c_p class because it will load in the driver and this tests for FPGA_NO_DRIVER on fpgaOpen. I could be wrong.

Reply:
This is correct. The test expects there to be no driver, so we do not want to setup like in the other fixture. This test is missing a comment header stating it's test case though.